users.json

```diff
@@ -1,16 +1,16 @@
 [
   {
-    "username": "cramakri",
-    "firstName": "Chandrasekhar",
-    "lastName": "Ramakrishnan",
-    "email": "cramakri@ethz.ch",
-    "password": "cramakri"
+    "username": "sdrenku65535",
```

Comment:
That's not funny enough, should be `C-3PO` and `R2-D2` with proper gravatars

Reply:
@leafty : (tms) == too much Star Wars...